### PR TITLE
72 backend edit locations functionality

### PR DESCRIPTION
**Pull request by ugedal1**
## Description

- Made message and handler so host is able to update locations.
- Locations now gets sent to players when they join the lobby, and when the host updates the locations.
- If no roles are given for a location all other players than the spy just get assigned the Player role
- 

### Type of change

- [x] New functionality 
- [x] Backend change

## Checklist:

- [x] My code follows the style guidelines of this project
- [ ] The solution is responsive (remove if not relevant)
- [x] I have performed a self-review of my code
- [x] I have commented my code, particularly in hard-to-understand areas
- [ ] I have made corresponding changes to the documentation
- [x] My changes generate no new warnings
- [ ] I have added tests that prove my fix is effective or that my feature works
- [ ] I have added tests for edge cases
- [x] New and existing unit tests pass locally with my changes

### CODE REVIEW

File: interloperServer/src/main/java/com/interloperServer/interloperServer/model/messages/outgoing/LobbyLocationsUpdateMessage.java [hunk 1]
```diff
@@ -0,0 +1,18 @@
...
+
+import java.util.List;
+
+import com.interloperServer.interloperServer.model.Location;
+
+public class LobbyLocationsUpdateMessage extends GameMessage {
+	private final List<Location> locations;
+
+	public LobbyLocationsUpdateMessage(List<Location> locations) {
+		super("LocationsUpdate");
```
Comment (supergelf) [inline interloperServer/src/main/java/com/interloperServer/interloperServer/model/messages/outgoing/LobbyLocationsUpdateMessage.java:11]: kanskje bruke liten bokstav siden alle andre event typer har det?

Comment (ugedal1) [reply interloperServer/src/main/java/com/interloperServer/interloperServer/model/messages/outgoing/LobbyLocationsUpdateMessage.java]: fixed!
